go-controller/go.mod

```diff
@@ -16,7 +16,6 @@ require (
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/imdario/mergo v0.3.8 // indirect
 	github.com/json-iterator/go v1.1.8 // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
```

Comment:
not sure what's up with this?

Reply:
I accidentally included this.  This gets dropped every time I build ovn-kubernetes ... in any case it's not relevant to this PR

Reply:
you still have this here. otherwise lgtm

Reply:
Looks like reflect2 gets pulled in by `vendor/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go` via `vendor/github.com/json-iterator/go`. So I think it's probably a legit dep?

Reply:
sorry, i'll remove this